salt/modules/aptly.py

```diff
@@ -50,6 +55,26 @@ def _cmd_run(cmd):
     return cmd_ret['stdout']
 
 
+def _convert_to_closest_type(value):
```

Comment:
This is generic enough that I wonder if we should put it in `salt.utils` so that other modules can use it.

Reply:
@cachedout At least for the conversion of strings to equivalent boolean values, I agree. I can submit a follow-up PR for that and modify the functionality here afterwards.